artwork/logo/coala-header.css

```diff
@@ -0,0 +1,24 @@
+.background{
+    background-color: #FFFFFF;
+}
+.coala{
+    float: right;
+    position: fixed;
+    top: 2.0vw;
+    right: 30.2vw;
+    color: #000000; 
+    font-family: 'Roboto';
+    overflow: hidden;
+    font-size: 7.2vw;
+}
+.io{
+    color: #7F7F7F; 
+    font-family: 'Roboto';
+}
+.logo-img{
+    position: fixed;
```

Comment:
The markup contains Bootstrap grid, but this is set as position fixed?

Or is this just for demonstration purpose? Does the logo still looks the same without this css?

Reply:
position: fixed caused the parent node doesn't have height. because it is the only content.
The height is usefull if you want to vertical align the next element. e.g the text `coala.io`. 

So you can stack the `logo` and `coala.io` right next to each other.